src/js/main.js

```diff
@@ -1,41 +1,47 @@
-import reqwest from 'reqwest'
 import mainHTML from './text/main.html!text'
 import share from './lib/share'
 import {pimpYouTubePlayer, getYouTubeVideoDuration} from './lib/youtube'
+import sheetToDOM from './lib/sheettodom'
 
 var shareFn = share('Interactive title', 'http://gu.com/p/URL', '#Interactive');
 
 
 export function init(el, context, config, mediator) {
-    el.innerHTML = mainHTML.replace(/%assetPath%/g, config.assetPath);
+const builder = document.createElement('div');
+builder.innerHTML = mainHTML.replace(/%assetPath%/g, config.assetPath);
 
-    [].slice.apply(el.querySelectorAll('.interactive-share')).forEach(shareEl => {
-        var network = shareEl.getAttribute('data-network');
-        shareEl.addEventListener('click', () => shareFn(network));
-    });
+    sheetToDOM('1NcSvccw77rHAncarwfeq7RGZF7yez9mP1Icf3oLMA7g', config.sheetName, builder, function callback(){
```

Comment:
Yes, it could be, though it is unlikely to change, I think it does make more sense in the config object.